core/kt-osrd-sim-infra/src/main/kotlin/fr/sncf/osrd/sim_infra/impl/RawInfraFromRjsBuilder.kt

```diff
@@ -87,11 +86,22 @@ class RawInfraFromRjsBuilderImpl : RawInfraBuilder {
         return trackSectionDistanceSortedChunkMap
     }
 
-    // TODO remove this accessor once useless in adapter
```

Comment:
why remove this note? The TrackNodeDescriptor object isn't supposed to be public, hence the comment about removing it

Reply:
This note was removed after choosing (debatable) to put the main part of the process in adapter instead of builder.
This requires read access to nodes [here](https://github.com/osrd-project/osrd/pull/6921/commits/30286b9651e73cea7e4efd1cf6b641499f2326c4#diff-0335c3a24d409feef2e7c3f5f3173cc4d08928c658e17faa8a0e71384922e6d3R303).
If after discussion this requirement is dropped, then I agree that this TODO must be back.

The debate on moving the process/changing builder's API will probably be [here](https://github.com/osrd-project/osrd/pull/6921#discussion_r1543028357).

Reply:
After discussion, the API is separated and the note is back: https://github.com/osrd-project/osrd/pull/6921/commits/754865f7833fe40281a8789e8f79108f55b6a2dc#diff-439a89c8361baafe08032d2f75202a6870d30ddea85b1f2c8e78754546798179R96